entity-web/src/main/java/eu/europeana/entity/web/config/EntityAppConfig.java

```diff
@@ -40,6 +56,12 @@ public MessageSource getMessageSource() {
         source.setDefaultEncoding("utf-8");
         return source;
     }
-    
+
```

Comment:
I see that this method is duplicated over APIs, could we move the method in commons.client-utils eventually by using a map as input, and we can easilly reuse it in APIs (Map.of). Probably the name of the method should be improved to something like create/buildAuthConfigProps